x-pack/test/functional/page_objects/reporting_page.js

```diff
@@ -175,7 +158,9 @@ export function ReportingPageProvider({ getService, getPageObjects }) {
 
     async setTimepickerInDataRange() {
       log.debug('Reporting:setTimepickerInDataRange');
```

Comment:
This was causing Discover to return quite a few items, hitting our byte limit. I shortened the timeframe to hopefully improve test speed + get our tests running again.